notebooks/conditional-generation.py

```diff
@@ -20,6 +21,15 @@
 ], columns=["RACE", "ETHNICITY", "GENDER"])
 
 
+# Configure Gretel credentials
+configure_session(
+    api_key="prompt",
+    cache="yes",
+    validate=True,
+    # default_runner="cloud",
+    # artifact_endpoint="cloud",
+)
```

Comment:
I don't have a strong feeling on how we format this specifically, but I do think it'd be nice to be consistent* across all our notebooks both in this repo and in the UCC (relational). As currently written, this presents additional optional arguments with their default values commented out. Would it be more useful to show what hybrid values would look like? Something like:
```python
configure_session(
    api_key="prompt",
    cache="yes",
    validate=True,
    # uncomment and complete below for hybrid contexts
    # default_runner="hybrid",
    # artifact_endpoint="s3://my-bucket", # replace with your bucket name
)
```


*Note: there may be cases where we're explicitly demoing how to configure a hybrid session, in which case obviously the presentation of `configure_session` will be different. In general, though, I think it'd be good to have a standard style for generic contexts.

Reply:
The majority of our notebooks today use one of the two following formats:
`configure_session(api_key="prompt", cache="yes", validate=True)`
or
`configure_session(endpoint="https://api.gretel.cloud", api_key="prompt", cache="yes")`

I think we should have the common format be the former - not show commented out defaults. And add explicit instructions in the docs about how to configure for hybrid (and maybe a text cell pointing those interested in hybrid to the docs)

@sami-t I'm curious if you have strong feelings on this

Reply:
+1 for keeping the original boilerplate. I'd optimize for keeping blueprint boilerplate consistent, concise and readable.

Getting a hybrid environment setup will require quite a bit of extra commitment and setup regardless, so I think asking these customers to update their boilerplate isn't too much of an ask. We'll also need to provide additional guidance such as configuring their local environment with access to CP object stores, so even adding the commented out config options isn't sufficient to setup a hybrid environment.